include/tf-psa-crypto/psa_build_info.h

```diff
@@ -1,5 +1,5 @@
 /**
- * \file tf-psa-crypto/build_info.h
+ * \file tf-psa-crypto/psa_build_info.h
```

Comment:
The "psa_" prefix does not seem the best choice here. We rather use it for what is directly related to the PSA crypto specification. Like we have `include/mbedtls/mbedtls_config.h`, it should probably rather be `include/tf-psa-crypto/tf_psa_crypto_build_info.h`.